lib/iris/tests/unit/fileformats/test_rules.py

```diff
@@ -105,22 +101,25 @@ def transform(cube):
 
 
 class TestLoadCubes(tests.IrisTest):
-    @tests.skip_biggus
     def test_simple_factory(self):
         # Test the creation process for a factory definition which only
         # uses simple dict arguments.
 
         # The fake PPField which will be supplied to our converter.
-        field = Mock()
-        field.data = None
+        field = mock.Mock()
+        field.core_data = mock.Mock()
+        # Add a compute attribute so that the data will be treated as lazy.
+        field.core_data.compute = None
```

Comment:
OK, I don't understand why the mock object needs to pretend to be lazy (the runs just fine without this line). I also don't see this line as a deal-breaker, though...